packages/zoe/test/unitTests/setupBasicMints.js

```diff
@@ -5,7 +5,7 @@ import { makeStore } from '@agoric/store';
 import { makeZoeKit } from '../../src/zoeService/zoe.js';
 import { makeFakeVatAdmin } from '../../tools/fakeVatAdmin.js';
 
-const setup = () => {
+export const setup = () => {
```

Comment:
I agree exporting at the declaration is better but I don't see it documented. Is there consensus to use it? I'm sure there's a lint rule we could enable.

Reply:
How do we find the lint rule?

Reply:
First look in ESilnt's built in rules:
https://eslint.org/docs/rules/
None applicable there. Then consider what plugins might have it. Here are the rules for eslint-plugin-import:
https://github.com/import-js/eslint-plugin-import/tree/main/docs/rules

There are two relevant rules:
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/exports-last.md
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/group-exports.md

Unfortunately they're to enforce the opposite of what we want. Often such rules can be configured per some preference to be "always" or "never" but these were designed less flexibly. A PR to make them more general would probably be accepted upstream. Meanwhile we could take take the code to make a custom rule.

I figure that's post MN-1 but if you're confident enough we should have this as a rule then let me know and I'll make a ticket.

Reply:
When discussing function styles --- `function` vs arrow function vs concise method syntax --- the pattern came up with a general sense of "this is good". But I wouldn't say we walked away with a sense of consensus on a decision. We did have consensus on the big one: arrow functions. The issue of patterns for using them were considered in weighing the tradeoffs.

In any case, I prefer it but I don't feel strongly about it. At such a time as we have a lint rule that could enforce this, we probably should. But until then, for this one, I'd just leave it as a preference rather than a rule.